x-pack/test/security_solution_api_integration/test_suites/entity_analytics/risk_engine/trial_license_complete_tier/risk_scoring_task/task_execution.ts

```diff
@@ -104,7 +104,7 @@ export default ({ getService }: FtrProviderContext): void => {
             );
           });
 
-          it('@skipInServerlessMKI starts the latest transform', async () => {
+          it('@skipInServerlessMKI @skipInServerless starts the latest transform', async () => {
```

Comment:
`@skipInServerless` is sufficient to exclude this test from both the MKI and PR pipelines. I left `@skipInServerlessMKI` to specify the scope of this exclusion more verbosely.